modular_skyrat/modules/tribal_extended/code/recipes.dm

```diff
@@ -0,0 +1,94 @@
+/datum/crafting_recipe/silkstring
+	name = "Silk String"
+	result = /obj/item/weaponcrafting/silkstring
+	time = 50
+	reqs = list(/obj/item/stack/sheet/cloth = 1)
+	category = CAT_MISC
+
+/datum/crafting_recipe/wood_bow
+    name = "Wooden Bow"
+    result = /obj/item/gun/ballistic/tribalbow
+    time = 300
+    reqs = list(/obj/item/stack/sheet/mineral/wood = 25,
+                /obj/item/weaponcrafting/silkstring = 5)
+    category = CAT_WEAPONRY
+
+/datum/crafting_recipe/pipebow
+	name = "Pipe Bow"
+	result = /obj/item/gun/ballistic/tribalbow/pipe
+	reqs = list(/obj/item/pipe = 5,
+				/obj/item/stack/sheet/plastic = 5,
+				/obj/item/weaponcrafting/silkstring = 5)
+	time = 450
+	category = CAT_WEAPONRY
+
+/datum/crafting_recipe/arrow
+	name = "Arrow"
+	result = /obj/item/ammo_casing/caseless/arrow/wood
+	time = 30
+	reqs = list(/obj/item/stack/sheet/mineral/wood = 1,
+				/obj/item/stack/sheet/cloth= 1,
```

Comment:
problem here is that with our ash walkers don't have cotton to grow to make cloth and with our new botany system dont have any stable way to get nutrience to feed cotton without it dying

Reply:
They have gauze from the medkit, as well as things they can scavenge from ruins and such. The original crafting recipes all involve silk and growing shit as well. Might make some cheap arrow alternative then?

Reply:
Honestly? It's fine as is. Arrows are meant to be in limited supply, and crafty Ashies may not have 30 of the highest tier arrows, but they can certainly get by with ammunition you can always pick up. I'm going to leave it as-is